### PR TITLE
DISTMONGO-303 Release notes 7.0.12

### PR DESCRIPTION
**Pull request by nastena1606**
new file:   docs/release-notes-v7.0.12.md
	modified:   docs/release-notes.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

**Review by sandraromanchenko — APPROVED** [review]

File: docs/release-notes-v7.0.12.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+for MongoDB.
+
+* *Percona Backup for MongoDB* is a distributed, low-impact solution for achieving
+consistent backups of MongoDB sharded clusters and replica sets.
+
+This release of Percona Distribution for MongoDB is based on the production release of [Percona Server for MongoDB 7.0.12-7](https://docs.percona.com/percona-server-for-mongodb/7.0/release_notes/7.0.12-7.html) and [Percona Backup for MongoDB 2.5.0](https://docs.percona.com/percona-backup-mongodb/release-notes/2.5.0.html).
+
+## Release Highlights
+
+* Upgraded the MozJS/SpiderMonkey engine that MongoDB uses to execute server-side JavaScript code within the database server to the latest ESR (Extended Support Release)
```
Comment (radoslawszulgo) [inline docs/release-notes-v7.0.12.md:20]: can we be specific to which version exactly?

File: docs/release-notes-v7.0.12.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* Made sure an old heartbeat that is being processed when primary catchup starts does not cause the primary catchup to think it's already caught up
```
Comment (radoslawszulgo) [inline docs/release-notes-v7.0.12.md:21]:
```suggestion
* Ensured that a replica set member's old heartbeat (ping) doesn't make the primary `catchUp` process mistakenly marked as complete.
```

**Review by radoslawszulgo — CHANGES_REQUESTED** [review]

File: docs/release-notes-v7.0.12.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Percona Distribution for MongoDB 7.0.12 (2024-07-18)
+
+[Upgrade now](installation.md){.md-button}
```
Comment (radoslawszulgo) [inline docs/release-notes-v7.0.12.md:3]:
```suggestion
[Upgrade now](minor-upgrade.md){.md-button}
```

File: docs/release-notes-v7.0.12.md (hunk 1 above)
Comment (nastena1606) [reply docs/release-notes-v7.0.12.md:20]: It's not stated by upstream:(

File: docs/release-notes-v7.0.12.md (hunk 3 above)
Comment (nastena1606) [reply docs/release-notes-v7.0.12.md:3]: Thanks!

**Review by radoslawszulgo — APPROVED** [review]

**Pull request merged**